lms/views/email.py

```diff
@@ -77,9 +78,12 @@ def preferences_redirect(self):
     )
     def preferences(self):
         return {
-            "preferences": self.email_preferences_service.get_preferences(
-                self.request.authenticated_userid
-            ).days(),
+            "jsConfig": {
+                "mode": JSConfig.Mode.EMAIL_NOTIFICATIONS,
```

Comment:
I'm not sure if importing `JSConfig` here is wrong. We could perhaps extract the `Mode` enum to use in `JSConfig` and here.

Reply:
I wouldn't bother with `JSConfig`. I would just do this:

```python
return {
    "jsConfig": {
        "mode": "email-notifications",
        "emailNotifications": self.email_preferences_service.get_preferences(
            self.request.authenticated_userid
        ).days(),
    }
}
```

I'd remove `JSConfig.Mode.EMAIL_NOTIFICATIONS` from the `JSConfig` class. There's just no need for these email preferences views to be entangled with that class.

Reply:
Perfect. I'll change it.